vector/base/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vector-deployment
+  labels:
+    app: vector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: vector
+  template:
+    metadata:
+      labels:
+        app: vector
+    spec:
+      containers:
+        - name: vector
+          image: timberio/vector:0.12.X-alpine
```

Comment:
Does this pull from a dockerhub? Can we change this or mirror the image to Quay please?

Reply:
Here ya go: https://quay.io/repository/operate-first/vector?tab=tags